CMSgov/hcgov-design-system
# [WNMGDS-1071] Add analytics to new header

### Added
- Added analytics tracking to the Header component using the core design system's analytics helper function

### Fixed
- Made `submenuTop` and `submenuBottom` optional in TypeScript

## How to test
I tested this in both an unauthenticated app and an authenticated app, and everything worked with minor adjustments to their props. If you'd like to test in another HC.gov app,
1. Change your `package.json` entry for `ds-healthcare-gov` to be a relative path to this repo, like `"@cmsgov/ds-healthcare-gov": "file:../hcgov-design-system",`
2. Remove `node_modules` and install dependencies again
3. Run the app after adjusting props
4. To test analytics specifically, you'll need to make sure utag scripts are loaded in development mode
